charts/erzwingungshaft/README.md

```
@@ -0,0 +1,24 @@
+```bash
+helm upgrade --install -f values.yaml erzwingungshaft charts/erzwingungshaft
+```
+
+The command deploys cronjob and additional configMap to start erzwingungshaft on kubernetes cluster. The [Configuration](#configuration) section lists the parameters that can be configured during installation. Use values.yaml confgMaps.ehaft to configure erzwingungshaft application.yaml.
+
+## Configuration
+
+| Key                                        | Type   | Default          | Description                                                                                                                                                                                                                                                |
+| ------------------------------------------ | ------ | ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| modules.ehaft.cronjob.schedule                           | string | - | Valid crontab expression to start kubernetes cronjob                                                                                                                                                                                                                                 |
+modules.ehaft.image                                  | object | -             | Image configuration (registry, repository, tag, pullPolicy)                                                                                                                                                                                                
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 192

(MD009, no-trailing-spaces)